src/ngx_http_lua_directive.c

```diff
@@ -1328,10 +1331,13 @@ ngx_http_lua_gen_chunk_name(ngx_conf_t *cf, const char *tag, size_t tag_len,
 
 found:
 
+    lmcf = ngx_http_conf_get_module_main_conf(cf, ngx_http_lua_module);
+    start_line = lmcf->directive_line ? : cf->conf_file->line;
```

Comment:
We should never use the GNU C language extension. Never. We should also add this check to the ngx-releng tool?